scripts/local_prometheus/local_node_exporter.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
When handled in the env these scripts can be deleted.

Reply:
Move them to compose here https://github.com/smartcontractkit/chainlink-node-compose/pull/7